index.bs

```diff
@@ -109,20 +200,33 @@ result or an error being returned to the [=local end=]. Multiple
 commands can run at the same time, and commands can potentially be
 long-running. As a consequence, commands can finish out-of-order.
 
-Each concrete [=command=] type is defined by:
- - A string <dfn export for=command>name</dfn>, implicit in its
-   definition.
- - A <dfn export for=command>parameter type</dfn>, which is a |map|
-   whose entries are the inputs to the command.
- - A set of [=remote end steps=].
- - A <dfn export for=command>return type</dfn>, which may be null.
+Each [=command=] is defined by:
+ - A <dfn export for=command>command type</dfn> which is defined by a [=remote
+   end schema=] fragment containing a group. Each such group has two fields:
+    - <code>method</code> which is a string literal of the form <code>[module
+      name].[method name]</code>. This is the <dfn export for=command>command
+      name</dfn>.
+    - <code>params</code> which defines a mapping containing data that to be passed into
+      the command. The populated value of this map is the
+      <dfn export for=command>command parameters</dfn>.
```

Comment:
Does this also need an `id` so that the local end can connect a response message to it?

Reply:
Yes, it does. It occurs to me that there are some details to work out here; I'm going to create an issue.